disnake/threads.py

```diff
@@ -873,6 +918,81 @@ async def delete(self, *, reason: Optional[str] = None) -> None:
         """
         await self._state.http.delete_channel(self.id, reason=reason)
 
+    async def add_tags(self, *tags: Snowflake, reason: Optional[str] = None) -> None:
+        """|coro|
+
+        Adds the given tags to this thread, up to 5 in total.
+
+        The thread must be in a :class:`ForumChannel`.
+
+        Adding tags requires you to have :attr:`.Permissions.manage_threads` permissions,
+        or be the owner of the thread.
+        Additionally, adding :attr:`~ForumTag.moderated` tags always requires said permissions.
```

Comment:
manage thread permissions
```suggestion
        However, :attr:`~ForumTag.moderated` tags may only be added by members with :attr:`.Permissions.manage_threads`.
```

Reply:
https://github.com/DisnakeDev/disnake/pull/726/commits/ca54244b7e6feec7309cc790ab189b8311a7abac